float32 nosewheelAngle
bool nosewheelShutdown
float32 propCommand
float32 precCommand
float32 finalAngle
float32 leftAngle
float32 rightAngle

